lib/crypto.js

```diff
@@ -562,15 +556,17 @@ exports.pbkdf2 = function(password,
 
 
 exports.pbkdf2Sync = function(password, salt, iterations, keylen, digest) {
```

Comment:
That would change the signature of `pbkdf2Sync` (it would accept the `callback` argument and change the value of `pbkdf2Sync.length`).